### ISSUE TITLE
Connection Refused - IOTA light Wallet

### ISSUE THREAD

**Issue by RamaDL** [iotaledger/wallet#558]
I'm trying to login in to my wallet and it returns "Connection Refused"

**Comment by wonkytonky** [comment]
change node

**Comment by RamaDL** [comment]
I tried with all nodes and it returns the same problem. I tried creating new seeds, login with and succeseded. The only account that i cant have access the one that have all my IOTA coins.

**Comment by wonkytonky** [comment]
@th0br0  i think you resovled this kind of issue before and a fix is in the works   please have a look

**Comment by ColDrekken** [comment]
Hi, im facing the same issue with IOTA Light Wallet 2.5.4 - IRI 1.4.1.2 . Tried all available nodes multiple times during the last days, but no luck.

**Comment by RamaDL** [comment]
I mine on mineiota.com, the only transactions that I received are from that page. I have around 120 transactions on my wallet history.

**Comment by ColDrekken** [comment]
@RamaDL yes, me too. around 96 transaction as i could look the last time and 4 adresses.

**Comment by O4siang** [comment]
#512 
Same issue here, I think the team will fix it soon.

**Comment by RamaDL** [comment]
Any solution? Not working... Still throwing Connection Refuse!

**Comment by jozmen97** [comment]
tengo el mismo problema alguna solucion

**Comment by onemoreitguy** [comment]
Same issue: connection refused after 124 transactions

**Comment by joeyabris** [comment]
I have the same issue.

**Comment by RamaDL** [comment]
@th0br0?

**Comment by jstillings1** [comment]
Try this
Change Node to custom and use this one.
https://iota.thathost.net:443
It has been the only node i can find that is working.

**Comment by onemoreitguy** [comment]
I don't think it's that's the issue.
Connecting to that or other host, I got the same pattern:
- I can connect (and use) two other wallets. 
- This wallet gave different errors: 'access denied', 'failed to connect to host' and takes a lot of time to login until it comes back to the login screen again.

**Comment by geot0** [comment]
> I can connect (and use) two other wallets.
This wallet gave different errors: 'access denied', 'failed to connect to host' and takes a lot of time to login until it comes back to the login screen again.

>

I have the exact same problems too

**Comment by onemoreitguy** [comment]
Here's the limitation of the nodes on the tangle
https://www.reddit.com/r/Iota/comments/7exwyq/psa_dont_use_mineiotacom_or_other_similar/

**Comment by ColDrekken** [comment]
@onemoreitguy thx for the link. 

But in the cases above everybody seems far away from the limit of 1000. Just round about 100 - 200 transactions so far.

**Comment by wonkytonky** [comment]
you can try the android app.. and see if you can login with the seed

**Comment by onemoreitguy** [comment]
android or desktop wallet version report the same.
@ColDrekken , I didn't look at the code, but the 1000 limit could include cascading between nodes and/or simultaneous wallet requests

**Comment by RamaDL** [comment]
When is released a new version of the wallet?

**Comment by posti85** [comment]
Same problem here. IOTA wallet is not a serious thing. IOTA team should not lie saying the technology is in beta, because it is clearly alfa.

**Comment by buragc** [comment]
I have the exact same issue. I had 3 transactions in my wallet. Waiting patiently for the new wallet but it is frustrating not being able to access my wallet. Any news on when the new version of the wallet will be released?

**Comment by joeyabris** [comment]
what shall we do?

**Comment by jurStv** [comment]
Still getting this error. When it will be resolved?

**Comment by onemoreitguy** [comment]
No luck so far on updates.
Trying to build a full sync node with the compiled workaround.
But takes quite some time :(

**Comment by onemoreitguy** [comment]
The issue reported by @RamaDL is not related to the last hours 'attack'.
It's a serious issue related to wallet with a significant amount of transactions and reattachs (ex: if you use mineiota).
I have 2 unaccessible wallets now for that and another reason.
At this stage I assume that both wallets are lost

**Comment by anomaly15** [comment]
Is there any information when the new wallet will be released to resolve this problem? I've had the issue for 2-3 weeks and can't access any coins to do anything.

**Comment by onemoreitguy** [comment]
I might have a temporary workaround, but I cannot solve it because I cannot find/connect to fully sync neighbor nodes :(

**Comment by vicman10000** [comment]
What can we do to help?

**Comment by onemoreitguy** [comment]
need a reliable full sync tcp node to connect to.
As long as my subtangle is not solid, the wallet fails to complete the load :(

**Comment by vicman10000** [comment]
If I set my light wallet as a full node would that help? Otherwise (if possible) could you provide reference, so that I could set it up.

**Comment by joserochadocarmo** [comment]
Change node? Does not work, even using node.dance site! 
Install and unistall wallet, Does not work!!
Install Iota Android, change pc or is a issue in your network. Does not work!
If you have many transactions...i have only 5 txs.

This is many of several advices, none works!!!! IT IS NOT A USER ISSUE!!!!

My seed doesnt work in wallet IOTA Light Wallet 2.5.4, and the unique place where i get
something was in smidgen/bitfinex lib. I see my pretty balance.

**Comment by joeyabris** [comment]
Miota price is skyrocketing.. Arrghhh, so much mixed feelings! We're doomed? Please devs help us..... :(

**Comment by c0ndensate** [comment]
I cannot log in either. Using IOTA Light Wallet 2.5.4 and my host is iota.bitfinex.com:80 

It will say LOGGING IN... and then after 15-20 minutes it says Connection Refused

**Comment by joeyabris** [comment]
We failed to connect because we made a lot of transactions in MINEIOTA.COM though.. We exceeded the limit of 1000 outgoing and incoming transactions. IMO 
I still have high hopes in IOTA foundation devs. Let us just wait for the next compliance.

**Comment by kryptoswim** [comment]
Help us 😔

**Comment by vicman10000** [comment]
Someone suggested using https://github.com/MichaelSchwab/iota-commandline-wallet

Any thoughts on the project?

**Comment by anomaly15** [comment]
@vicman10000 Just letting you know I've gone through the steps but unfortunately even command line does not work. It looks like it still relies on running a full node as you have to edit java files to increase memory limit. Found here: https://iotatangle.slack.com/files/U1VE8FAPN/F2A9HCV28/current_workaround_for_connection_refuse.txt - Unfortunately I'm not great enough to understand Java enough to customize and I can't find any guides to get into that part of it.

**Comment by onemoreitguy** [comment]
The issue is the node/client don't currently handle wallets with large transactions

The 'connection refused' is a standard error response to the client based on the real server response to the transaction limit exceed (you can see it using a wireshark http trace)

I already managed to build a 'read-access-synced' (transactions not allowed) node, saw my wallets and tested successfully with other stressed users

If you kept your seeds private everything is safe. Just wait for a newer version of the wallet.

just to clarify everyone:
-     I'm not part of the dev team. Just a project enthusiast and follower as an IT guy
-     the dev team has no access to your wallets or seeds (I hope not !).

**Comment by ThomasPepperz** [comment]
For impending transactions taking forever, Try configuring your node as
attached.

Then, identify the earliest transaction you created (if you did more than
one or tried, if you didn't let me know your situation)

Reattach the earliest pending transaction with positive balance first,
thenrebraodcast. go to receive. generate an address and let your computer
do some work. Then, chill and wait while. Let me know if this works or if
doesn't. You can follow me on twitter for postings in regard to IOTA, the
wallet, and a troubleshoot guide I am trying to compile this weekend.

**Comment by ThomasPepperz** [comment]
For impending transactions taking forever, Try configuring your node as
attached.

Then, identify the earliest transaction you created (if you did more than
one or tried, if you didn't let me know your situation)

**Comment by onemoreitguy** [comment]
I'm missing something, or it already to late in the evening, but how to:
 "...configuring your node as attached"

**Comment by willshu2049** [comment]
Same situation. Has anybody been able to log into their account?

**Comment by joeyabris** [comment]
still.. I can't log in

**Comment by onemoreitguy** [comment]
Just to summarize and let everyone to understand:
ISSUE: users are getting connection error when trying to sign-in
CAUSE: wallets with a significant amount of transactions are disconnected after awhile by the nodes. There is a limit defined that prevents large amount of data transfers (max:1000).
WORKAROUND: (a) install a full node (or find one). (b) Define on the ini file the following parameter MAX_REQUESTS_LIST=10000. (c) connect the wallet. It takes more than 10-60 minutes for the node and the wallet to process all the records
SOLUTION: wait for a node and wallet releases

Again: I'm not related to the dev team (I'm just a IOTA supporter and enthusiast) and I found this by searching and code reading

**Comment by joserochadocarmo** [comment]
Ok you right panda. But where come all this amount transactions related to my seed, i just made 5 transactions! But I'm waiting. Thanks

**Comment by onemoreitguy** [comment]
Those transactions you need to show me.
I have a feeling, but I cannot guess ;)

**Comment by Ps3ud-0nym** [comment]
I've made one small transaction and this is happening to me, so..

**Comment by pkkirilov** [comment]
same problem here, only a few transactions...

**Comment by ukmarn** [comment]
@pkkirilov @Ps3ud-0nym @joserochadocarmo @onemoreitguy @joeyabris @willshu2049 @anomaly15 @vicman10000 
I was having same problem as mentioned above, with less than 70 transactions. Few hours ago found a thread on github regarding same problem, with a link to website holding information of IOTA nodes. And guess what, after trying every available node inside IOTA wallet and getting "Connection Refused", I decided to try the new nodes out and it worked lke a charm! It usually takes a bit longer to log in but at least I managed to access my balance.
I think you should try it out, hope it works for you too.

Link to list of nodes: https://iotasalad.org/nodes
Node that worked for me: https://potato.iotasalad.org:14265

**Comment by joeyabris** [comment]
@ukmarn It's working man!! I love you man! hahahaha! I am very fine now. Thanks though! :))
worked for me - https://potato.iotasalad.org:14265

**Comment by atolby** [comment]
![light-wallet_confirm](https://user-images.githubusercontent.com/25606511/33805415-cea0ad22-ddb8-11e7-8d94-03b6b295784f.jpg)

**Comment by chris0r** [comment]
Hi, it worked for me as well and I have access to the wallet but now I can't transfer the IOTA to another Seed and Address. I tried it with 1 IOTA 20h ago and its still pending despite multiple rebroadcast and reattaching. 

Can me someone help.

Many thx

**Comment by anomaly15** [comment]
@chris0r Yeah I have the exact same problem but my transaction has been pending for 5 days now. I understand it's because the node is at transaction capacity so we're queued up with a bunch of others. It was recommended to move to another node but we can't because potato is the only one that will allow seed login with connection refused error. So we are stuck on a rock in a hard place.

All we can hope is that IOTA team will place more priority on a working wallet and nodes. It's currently not capable of handling any transaction traffic so I wouldn't suggest sending any miota anywhere.

**Comment by taylankasap** [comment]
I was having the same issue. Now I can login with the wallet version v2.5.5

**Comment by bberish** [comment]
Anyone able to connect to the wallet with the new update? I am attempting to login, but it has been "logging in" for the past 20 mins now

**Comment by simpleifreak** [comment]
same issue for me, transaction from binance already prooved, but i cant login at light wallet, connection refused , all the same. please help me, guys

**Comment by joshivince** [comment]
Same issue here. Connection refused. I also had mined (for 2 days only) at mineiota. Had only 2 or 3 transactions. With https://potato.iotasalad.org:14265 as a custom node it worked like a charm.

Thank you!

**Comment by mchoeti** [comment]
same here , Connection refused, since today. i tried a couple of different notes but without success.
